client/webpack.server.rails.config.js

```diff
@@ -0,0 +1,40 @@
+// Webpack configuration for server bundle
+
+const webpack = require('webpack');
+const path = require('path');
+
+const devBuild = process.env.NODE_ENV !== 'production';
+const nodeEnv = devBuild ? 'development' : 'production';
+
+module.exports = {
+
+  // the project dir
+  context: __dirname,
+  entry: ['./app/bundles/HelloWorld/startup/serverGlobals'],
+  output: {
+    filename: 'server-bundle.js',
+    path: '../app/assets/javascripts/generated',
+  },
+  resolve: {
+    extensions: ['', '.webpack.js', '.web.js', '.js', '.jsx', 'config.js'],
+    alias: {
+      lib: path.join(process.cwd(), 'app', 'lib'),
+    },
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
```

Comment:
@robwise @yorzi, I believe @josiasds is right and we have already fixed this. @robwise Please feel free to merge @yorzi's change to master and regenerate.